conf.py

```diff
@@ -34,6 +34,7 @@
     'cylc.sphinx_ext.diff_selection',
     'cylc.sphinx_ext.grid_table',
     'cylc.sphinx_ext.hieroglyph_addons',
+    'cylc.sphinx_ext.metadata',
```

Comment:
I think it would probably make more sense to put this in `cylc.sphinx_extensions.cylc_lang.autodocumenters`.

(to tackle the GPL license issue we would be more likely to move the other autodocumenter out into cylc-doc since it is single-purpose).

Reply:
> the other autodocumenter

The global/platform doc?

Reply:
The "other" autodocumenter (the only one in the `autodocumenters.py` file) is the one we use to document the `SPEC` objects (i.e. the configuration schema).

Reply:
Wait a minute - are you suggesting that the current approach is fine?

Reply:
I'm saying move your autodocument into `cylc.sphinx_extensions.cylc_lang.autodocumenters` with the others rather than creating a new extension for it.